modules/base/rendering/screenspacerenderablerenderable.cpp

```diff
@@ -0,0 +1,294 @@
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/base/rendering/screenspacerenderablerenderable.h>
+
+#include <openspace/camera/camera.h>
+#include <openspace/rendering/renderable.h>
+#include <openspace/scene/rotation.h>
+#include <openspace/scene/scale.h>
+#include <openspace/scene/translation.h>
+#include <openspace/util/updatestructures.h>
+
+namespace {
+    constexpr openspace::properties::Property::PropertyInfo TimeInfo = {
+        "Time",
+        "Time",
+        "The time (in J2000 seconds) that is used to calculate transformations and the "
+        "renderable's data."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraPositionInfo = {
+        "CameraPosition",
+        "Camera Position",
+        "Specifies the location of the virtual camera that is showing the renderable "
+        "class. This position is provided in meters."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraCenterInfo = {
+        "CameraCenter",
+        "Camera Center",
+        "The location of the camera's focal point. The camera's view direction will "
+        "always be pointing at the provided center location. This position is provided "
+        "in meters."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraUpInfo = {
+        "CameraUp",
+        "Camera Up",
+        "The direction that is 'up' for the provided camera. This value does not have "
+        "any units."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraFovInfo = {
+        "CameraFov",
+        "Camera Field-of-view",
+        "The camera's field-of-view in degrees."
+    };
+
+    const openspace::properties::PropertyOwner::PropertyOwnerInfo TransformInfo = {
+        "Transform",
+        "Transform",
+        "The Translation, Rotation, and Scale that are applied to the rendered "
+        "Renderable."
+    };
+
+    struct [[codegen::Dictionary(ScreenSpaceRenderableRenderable)]] Parameters {
+        std::optional<std::string> identifier [[codegen::private()]];
+
+        // The [Renderable](#renderable) object that should be shown in this ScreenSpace
```

Comment:
```suggestion
        // The [Renderable](#renderable) object to show in this ScreenSpace
```
Just shorter and more straightforward => easier to read 

I'll comment on the other descriptions as well, but leave the re-formatting for line width up to you